src/Education/XamlGenerator/IRuleHelpXamlBuilder.cs

```diff
@@ -166,9 +164,8 @@ private void ProcessElement()
                     break;
 
                 case "blockquote":
-                    writer.WriteStartElement("Section");
-                    writer.WriteAttributeString("Margin", "16,0,0,0");
-                    writer.WriteAttributeString("Background", "LightGray");
```

Comment:
I've removed all of the hard-coded styling properties and replaced them with references to `Style` resources. That allows the hosting control to change the look without us needing to change the generator.

The hosting control will contain something like the following (which I'll add in the next PR):

```
    <UserControl.Resources>
        <Style x:Key="Pre" TargetType="Section">
            <Setter Property="TextAlignment" Value="Left"/>
            <Setter Property="Foreground" Value="{DynamicResource {x:Static vsshell:VsBrushes.CaptionTextKey}}"/>
            <Setter Property="FontStyle" Value="Italic"/>
            <Setter Property="FontFamily" Value="Courier New"/>
        </Style>
        <Style x:Key="Heading2" TargetType="Paragraph">
            <Setter Property="FontFamily" Value="{DynamicResource {x:Static vsshell:VsFonts.CaptionFontFamilyKey}}"/>
            <Setter Property="FontSize" Value="{DynamicResource {x:Static vsshell:VsFonts.Environment122PercentFontSizeKey}}"/>
            <Setter Property="FontWeight" Value="{DynamicResource {x:Static vsshell:VsFonts.Environment122PercentFontWeightKey}}"/>
        </Style>
    </UserControl.Resources>
```